java/connector-node/risingwave-sink-es-7/src/main/java/com/risingwave/connector/EsSink.java

```diff
@@ -198,13 +191,9 @@ public EsSink(EsSinkConfig config, TableSchema tableSchema) {
         this.bulkProcessor = createBulkProcessor(this.requestTracker);
 
         primaryKeyIndexes = new ArrayList<Integer>();
```

Comment:
The `primaryKeyIndexes` should not be used any more and should be removed.

Reply:
If we still need a schema from the upstream, why we don't need pk index, even though the schema is (varchar, jsonb) in most cases.

Reply:
In this PR, all processing is handled in rust side, including generating the doc id and processing the content body. On java side we just build and send request from the processed doc id and content body, so the pk index is not used on java side.